html/styles.css

```diff
@@ -0,0 +1,378 @@
+html,
+body {
+  padding: 0;
+  margin: 0;
+  height: 100%;
+}
+
+
+body {
+  font-family: Arial, Helvetica, sans-serif, serif;
+}
+
+.page-header {
+  position: relative;
+  display: block;
+  height: 400px;
+  overflow: hidden;
+  background-image: url('../images/header-image.jpg');
+}
+
+.page-header .header-logo {
+  height: 100px;
+  margin-left: 100px;
+  margin-top: 20px;
+  margin-bottom: 0;
+  background-image: url('../images/adoptadog-logo.svg');
+  background-repeat: no-repeat;
+}
+
+.page-header .checkout-balance {
+  height: 30px;
+  color: white;
+  position: absolute;
+  top: 20px;
+  right: 100px;
+}
+
+.page-header .checkout-balance .cart-icon {
+  width: 20px;
+  margin-right: 2px;
+}
+
+.page-header .checkout-balance .dollar {
+  font-weight: 600;
+  font-size: 8pt;
+}
+
+.page-header .main-title {
+  margin-left: 100px;
+  margin-top: 0;
+  font-family: sans-serif;
+  font-size: 14pt;
+  color: #81B841;
+}
+
+.page-header .button-checkout {
+  background-color: #81B841;
+  border: none;
+  color: white;
+  padding: 5px 20px;
+  font-size: 7pt;
+}
+
+.page-header .button-checkout {
+  position: absolute;
+  right: 95px;
+  top: 50px;
+  text-transform: capitalize;
+  text-decoration: none; 
+}
+
+.page-header .button-checkout:hover {
+  opacity: 0.9;
+}
+
+.topnav {
+  background-color: #e0e0e0;
+  padding: 20px 10%;
+  font-size: 10pt;
+  
+  font-weight: bold;
+  word-spacing: 5px;
+}
+
+.navigation-link {
+  text-decoration: none;
+  color: #134350;
+  font-weight: 800;
+}
+
+main {
+  width: 80%;
+  margin: auto;
+  padding-bottom: 50px;
+}
+
+.introduction {
+  display: flex;
+  align-items: flex-start;
+  margin: 20px 0;
+}
+
+.introduction .intro-image {
+  width: 35%;
+  margin-right: 40px;
```

Comment:
I would look and see if you can make your image a bit larger here to better match the website mockup.